### PR TITLE
add display-mode: picture-in-picture example and fix sizing issue

### PR DESCRIPTION
**Pull request by chrisdavidmills**
Chrome 123 adds support for the [`@media` `display-mode`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/display-mode) `picture-in-picture` value. This PR adds CSS to our document picture-in-picture example that combines `display-mode: picture-in-picture` with `prefers-color-scheme` to provide a light and dark color scheme just for the picture-in picture window.

- ChromeStatus: https://chromestatus.com/feature/5198101675769856
- Spec: https://drafts.csswg.org/mediaqueries-5/#display-mode-picture-in-picture
- Basic demo: https://document-picture-in-picture-api.glitch.me/display-mode.html

### CODE REVIEW

**Comment by chrisdavidmills** [comment]
@pepelsbey, would you mind giving this little demo tweak a review?

File: document-picture-in-picture/main.css [hunk 1]
```diff
@@ -11,3 +11,21 @@
...
+		background: antiquewhite;
+
+	}
+}
+
+@media (display-mode: picture-in-picture) and (prefers-color-scheme: dark) {
+	body {
+		background: #333;
+
+	}
```
Comment (pepelsbey) [inline document-picture-in-picture/main.css:26]:
```suggestion
@media (display-mode: picture-in-picture) and (prefers-color-scheme: light) {
	body {
		background: antiquewhite;
	}
}

@media (display-mode: picture-in-picture) and (prefers-color-scheme: dark) {
	body {
		background: #333;
	}
```

**Review by pepelsbey — APPROVED** [review]
I’d suggest cleaning up the formatting. Otherwise, looks good to me :) Thanks!

**Comment by chrisdavidmills** [comment]
@pepelsbey thanks! Are you able to merge it? I don't have permission to do so in this repo.

**Pull request merged**